axelor-redmine/src/main/resources/data-init/input/base_appRedmine.csv

```diff
@@ -1,2 +1,2 @@
-"name";"code";"installOrder";"description";"imagePath";"modules";"dependsOn";"sequence";"company.importId";"redmineProjectInvoiceable";"redmineProjectClientPartner";"redmineProjectInvoicingSequenceSelect";"redmineIssueProduct";"redmineIssueIsTaskRefused";"redmineIssueDueDate";"redmineIssueEstimatedTime";"redmineIssueInvoiced";"redmineIssueAccountedForMaintenance";"redmineTimeSpentProduct";"redmineTimeSpentDurationForCustomer"
-"Redmine";"redmine";30;"Redmine configuration";"app-redmine.png";"axelor-redmine";"project,helpdesk";53;1;"Invoiceable";"Customer code";"Invoicing Type";"Product";"Prestation refusée/annulée";"Date d'échéance (INTERNE)";"Temps estimé (INTERNE)";"Déjà facturé";"Comptabilisé maintenance";"Product";"Temps passé ajusté client"
+"name";"code";"installOrder";"description";"imagePath";"modules";"dependsOn";"sequence";"company.importId";"redmineProjectInvoiceable";"redmineProjectClientPartner";"redmineProjectInvoicingSequenceSelect";"redmineIssueProduct";"redmineIssueIsTaskRefused";"redmineIssueDueDate";"redmineIssueEstimatedTime";"redmineIssueInvoiced";"redmineIssueAccountedForMaintenance";"redmineTimeSpentProduct";"redmineTimeSpentDurationForCustomer";"redmineIssueIsTaskAccepted";"redmineIssueIsOffered";"redmineIssueUnitPrice";"redmineTimeSpentActivityTypeSelect"
```

Comment:
no need of redmineTimeSpentActivityTypeSelect as it is not mapped with custom field of redmine
remove it